run-clang-format.py

```diff
@@ -298,6 +301,9 @@ def main():
         default=[],
         help='exclude paths matching the given glob-like pattern(s)'
         ' from recursive search')
+    parser.add_argument(
+        '-style',
+        help='formatting style to apply (LLVM, Google, Chromium, Mozilla, WebKit)')
```

Comment:
Also style can be `file` to use the .clang-format file

Reply:
It works when entering the format file. Please tell me if you have any problems.